apiserver/facades/agent/uniter/uniter_legacy_test.go

```diff
@@ -355,12 +355,13 @@ func (s *uniterLegacySuite) TestConfigSettings(c *gc.C) {
 }
 
 func (s *uniterLegacySuite) TestCurrentModel(c *gc.C) {
-	model := s.ControllerModel(c)
 	result, err := s.uniter.CurrentModel(context.Background())
 	c.Assert(err, jc.ErrorIsNil)
+	modelInfo, err := s.ControllerDomainServices(c).ModelInfo().GetModelInfo(context.Background())
+	c.Assert(err, jc.ErrorIsNil)
 	expected := params.ModelResult{
-		Name: model.Name(),
-		UUID: model.UUID(),
+		Name: modelInfo.Name,
+		UUID: s.ControllerModelUUID(),
```

Comment:
Make this `modelInfo.UUID` - the intent is much clearer. It's not immediately obvious that modelInfo is read from the controller model

Reply:
Done & Done